### DISCUSSION TITLE
Please remove very old apps from the Software Manager?

### DISCUSSION THREAD

**Discussion by dharmabum5** [linuxmint/discussions#97] [Pull requests]
Hi,

Can Linux Mint please remove very old apps from the Software Manager? Say, anything that hasn't been used/downloaded since 5 years and older. 

Is this the right place for this request? If not, please provide a link.

**Comment by Secret-chest** [comment]
The LM team doesn't control what the Software Manager contains. Debian, Ubuntu and Flathub control it by default. The only apps Linux Mint packages are http://packages.linuxmint.com/list.php?release=victoria.

**Comment by Jopp-gh** [comment]
I agree with the OP.

The benefit to filter (not remove!) old software would be to protect users from installing potentially breaking software components. Quality control should not be that lax! 

Developers could add a checkbox to filter depreciated software.

This way you give the end user the responsibility and preference to install older software, if they really need older packages and know what they are doing.
